### DISCUSSION TITLE
Dawn/Day buffer

### DISCUSSION THREAD

**Discussion by kme77** [StealthChesnut/HA-FoxESS-Modbus#70] [Q&A] [answered]
I'm probably just being an idiot but could you expand on how the dawn/Day buffer works. I've been messing around with it but I'm still unsure?

I assumed that it reserved the amount charge you set so it never dropped below that amount, is that correct? If so I think I understand the dawn buffer (i.e. overnight it won't drop below that charge before dawn) but how is the day buffer used and how do the two  buffers interact?
Cheers!

**Comment by Daibutt** [comment, marked as answer]
I don't think that is part of this integration?

**Reply by kme77** [reply]
Ahh yes sorry, I posted in the wrong tab! 😁
